detectron2/modeling/roi_heads/mask_head.py

```diff
@@ -234,7 +241,7 @@ def __init__(self, input_shape: ShapeSpec, *, num_classes, conv_dims, conv_norm=
                 padding=1,
                 bias=not conv_norm,
                 norm=get_norm(conv_norm, conv_dim),
-                activation=F.relu,
+                activation=nn.ReLU(),
```

Comment:
this change is not necessary

Reply:
if this change is not conducted, Runtime Error will be raised when call `torch.jit.script(model)`:
```
RuntimeError: 
function cannot be used as a value:
  File "detectron2/layers/wrappers.py", line 107
        if self.norm is not None:
            x = self.norm(x)
        if self.activation is not None:
           ~~~~~~~~~~~~~~~ <--- HERE
            x = self.activation(x)
        return x
```
if there better solution to handle this?

Reply:
ok, let's keep it like this